.eslintrc.json

```diff
@@ -1,13 +1,11 @@
 {
-    "env": {
-        "es6": true,
-        "node": true
-    },
-    "extends": "wordpress",
-    "rules": {
-        "indent": [
-            "error",
-            "tab"
-        ]
-    }
+  "env": {
+    "es6": true,
+    "node": true
+  },
+  "extends": "plugin:@wordpress/eslint-plugin/recommended",
+  "rules": {
+    "one-var": ["error", "never"],
+    "prefer-arrow-callback": ["error"]
+  }
 }
```

Comment:
Instead of using a JSON file for the config, lets switch it to a JS file `.eslintrc.js` for parity with Gutenberg, see https://github.com/WordPress/gutenberg/blob/master/.eslintrc.js